packages/compass-components/src/components/actions/item-action-controls.tsx

```diff
@@ -39,7 +33,7 @@ export type ItemActionControlsProps<Action extends string> = {
   'data-testid'?: string;
 };
 
-export function ItemActionControls<Action extends string>({
+export function ItemActionControls<Action extends string = string>({
```

Comment:
TIL that you can have a default

Reply:
Yeah – it usually just reduces type safety, though, because then consumers are likely to omit the type (and the default is typically the most generic type available).

This also seems to be one of those cases where type safety goes away – I don't seen an obvious reason for that in the rest of the code, though, so maybe in this particular case we could reconsider?

Reply:
🙂 TBH, it's likely not strictly needed here, since we'll always be using this [in a way that can infer the type of this type parameter from the `props`](https://www.typescriptlang.org/play/?#code/C4TwDgpgBAYg9nACgJzmAzgHgEIENkB8UAvFAN5QBG+AXFHslAL4DcAUACYQDGANvtABmAVwB23YAEs4o2AkwAVKBAAewCKI7oo6YMkmiA5gQAUYVBjrwkFrAoIBKOqEhxBUc2nQBtAETVkXwBddm4ZXShkCHRhXmASOTgTCgC6XxA4XyhcbTDRCKYHdgB6YqhyqAA9AH42IA).